docs/client-metrics.md

```diff
@@ -206,12 +206,17 @@ The event stream is a log of events and the time they occurred while the user is
 * signup.email-optin.visible.false - email opt-in is not visible
 * signup.checkbox.change.marketing-email-optin.checked - user checked the email opt-in
 * signup.checkbox.change.marketing-email-optin.unchecked - user unchecked the email opt-in
-* signup.checkbox.change.customize-sync.checked - user checked the "Choose what to sync" checkbox.
-* signup.checkbox.change.customize-sync.unchecked - user unchecked the "Choose what to sync" checkbox.
+* signup.checkbox.change.customize-sync.checked - user checked the "Choose what to sync" checkbox
+* signup.checkbox.change.customize-sync.unchecked - user unchecked the "Choose what to sync" checkbox
 * error.signup.auth.1012 - user did not enter year of birth
 * error.signup.auth.1016 - user did not enter their birthday
 * error.signup.auth.1029 - signup is disabled on Fx for iOS v1
-* error.signup.auth.1030 - signup has been force disabled by a Sync based relier.
+* error.signup.auth.1030 - signup has been force disabled by a Sync based relier
+* experiment.pw-strength.ALL_NUMBERS_LETTERS - password contains all numbers or letters
+* experiment.pw-strength.BLOOMFILTER_HIT - password was found in bloomfilter
+* experiment.pw-strength.BLOOMFILTER_MISS - password was not found in bloomfilter
+* experiment.pw-strength.PASSWORD_TOO_SHORT - password was less than 8 characters
+* experiment.pw-strength.MISSING_PASSWORD - no password supplied
 
```

Comment:
I removed the period at the end of the sentences, so that they would be consistent throughout the signup portion.